spec/TwitchApi/Resources/EventSubApiSpec.php

```diff
@@ -14,20 +14,26 @@ class EventSubApiSpec extends ObjectBehavior
     private string $secret = 'SECRET';
     private string $callback = 'https://example.com/';
 
-    private function createEventSubSubscription(string $type, string $version, array $condition, RequestGenerator $requestGenerator)
+    private function createEventSubSubscription(string $type, string $version, array $condition, RequestGenerator $requestGenerator, bool $isBatchingEnabled = null)
     {
         $bodyParams = [];
 
         $bodyParams[] = ['key' => 'type', 'value' => $type];
         $bodyParams[] = ['key' => 'version', 'value' => $version];
         $bodyParams[] = ['key' => 'condition', 'value' => $condition];
-        $bodyParams[] = ['key' => 'transport', 'value' => [
-          'method' => 'webhook',
-          'callback' => $this->callback,
-          'secret' => $this->secret,
-          ]
+        $bodyParams[] = [
+            'key' => 'transport',
+            'value' => [
+                'method' => 'webhook',
+                'callback' => $this->callback,
+                'secret' => $this->secret,
+            ],
```

Comment:
just rewrite in format that we  using in original class

Reply:
Great catch! Thank you for your keen eye.